### PR TITLE
Sunrise Examples and Sphinx Documentation

### PR DESCRIPTION
**Pull request by nsryan2**
I think I included all of the right files in this commit but if not please let me know. As these files are, there's one warning and it's that modules.rst isn't included in the documentation. I also don't know how attracted you were to alabaster, but I changed the theme to [cloud](https://pypi.org/project/cloud_sptheme/) (there are a lot of other options if you don't like it).

### CODE REVIEW

**Review by samgdotson — CHANGES_REQUESTED** [review]
Great job generating documentation and examples! I was able to build the docs and they look great. A few minor things before I can merge:
1. The files in ``_build`` are built files. If I run the ``Makefile`` then that folder should be generated. Correct? So there's no need to include them in the PR. The PR should only include the files necessary to build the docs on my own (or anyone else for that matter). 
2. Add some instructions to the ``README`` explaining how to build the files. I had to install the cloud theme before the makefile worked. Some people may not have sphinx installed, either. A short explanation is useful.
3. Add necessary packages to ``requirements.txt`` (only if they can be added via ``pip install``). 
4. There are several warnings about "Unexpected section titles". Is there a way to resolve these?

Edit: If you determine that some filetypes don't need to be in the PR for the docs to build, consider adding it to the ``.gitignore`` (you can see it by doing ``ls -A`` on linux, or simply open it in your code editor). 
E.g. If no ``.html`` files are needed to build the docs, try adding the line ``*.html``.

File: docs/_build/html/_sources/examples.rst.txt [hunk 1]
```diff
@@ -0,0 +1,357 @@
...
+>>> hour = hour_number(3,0.1)
+>>> equation_of_time(hour, True)
+    -4.684279708368122
+    
+For a period of two years, the function equation_of_time behaves as
+
+>>> w = np.linspace(0,17520,87600).tolist()
+>>> ts = []
+>>> for i in w:
+>>>     ts.append(equation_of_time(i))
```
Comment (samgdotson) [inline docs/_build/html/_sources/examples.rst.txt:150]:
```suggestion
>>> ts = [equation_of_time(i) for i in w]
```
This can be one line with python list comprehension.

File: docs/_build/html/_sources/examples.rst.txt [hunk 2]
```diff
@@ -0,0 +1,357 @@
...
+>>> hour = hour_number(3,0.1)
+>>> declination(hour, True)
+    -22.81293175279647
+
+For a period of one year, the function declination behaves as
+
+>>> w = np.linspace(0,8760,87600).tolist()
+>>> ts = []
+>>> for i in w:
+>>>     ts.append(declination(i))
```
Comment (samgdotson) [inline docs/_build/html/_sources/examples.rst.txt:120]:
```suggestion
>>> ts = [declination(i) for i in w]
```
This can be one line with list comprehension in python.

I know there are others, you might consider changing.

**Review by samgdotson — CHANGES_REQUESTED** [review]

**Comment by nsryan2** [comment]
Thanks for the feedback! I've added instructions and requirements to the readme and requirements file. All files in the _build folder should now be ignored in future commits, and I got rid of the Unexpected Section title warnings. Turns out numpy docstrings aren't supposed to have a colon after them. I resolved the conflicts with the .gitignore but I'm not entirely sure where all of them came from so you might want to check that (unless you recently added a bunch of extensions to it).

File: docs/examples/examples_sunrise.rst [hunk 3]
```diff
@@ -0,0 +1,344 @@
...
+>>> ts = [equation_of_time(i) for i in w]
+>>> plt.plot(w,ts)
+>>> plt.title('Equation of Time')
+>>> plt.xlabel('Hour Number')
+
+.. image:: ../examples/plots/sunrise_plots/et_2year.png
+    :align: center
+
+
+:py:func:`local_meridian`
```
Comment (samgdotson) [inline docs/examples/examples_sunrise.rst:157]:
```suggestion
:py:func:`sunrise.local_meridian`
```
Minor error. Changed for consistency with the other functions.

File: sunrise.py [hunk 4]
```diff
@@ -153,22 +175,22 @@ def local_meridian(utc=utc):
     LSTM. The LSTM is a reference meridian for given time zones.
     LSTM is calculated according to the equation:
         LSTM = 360/24*UTC
+
```
Comment (samgdotson) [inline sunrise.py:178]:
```suggestion

```
pep8 fix (literally just removing unnecessary whitespace).

**Review by samgdotson — CHANGES_REQUESTED** [review]
It looks much better. I made a couple of style suggestions. I actually didn't know that the numpy style doesn't include a colon! Good to know!

Thanks for all of your hard work! Let me know when you've made these two changes and I'll merge.

**Review by samgdotson — APPROVED** [review]
LGTM
